CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+Since 1.2
+=========
+
+  * #1044 creates a `Kernel` struct with a method for the kernel's main loop,
```

Comment:
It doesn't like like GH auto-links numbers in generic markdown files :(
https://github.com/tock/tock/blob/kernel-oop-just-work/CHANGELOG.md